src/metatrain/soap_bpnn/trainer.py

```diff
@@ -34,6 +34,8 @@
 from metatrain.utils.transfer import (
     systems_and_targets_to_device,
     systems_and_targets_to_dtype,
+    tensormap_to_device,
+    tensormap_to_dtype,
```

Comment:
Why are these separate instead of a single `batch_to` that moves a full batch to dtype and device?